_translations/en.yml

```diff
@@ -31,12 +31,8 @@ en:
     title: "Bitcoin for Businesses - Bitcoin"
     pagetitle: "Bitcoin for Businesses"
     summary: "Bitcoin is a very secure and inexpensive way to handle payments."
-    lowfee: "The lowest fees out there"
-    lowfeetext: "Bitcoin's high cryptographic security allows it to process transactions in a very efficient and inexpensive way. You can make and receive payments using the Bitcoin network with almost no fees. In most cases, fees are not strictly required but they are recommended for faster confirmation of your transaction."
     fraud: "Protection against fraud"
     fraudtext: "Any business that accepts credit cards or PayPal knows the problem of payments that are later reversed. Chargeback frauds result in limited market reach and increased prices, which in turn penalizes customers. Bitcoin payments are irreversible and secure, meaning that the cost of fraud is no longer pushed onto the shoulders of the merchants."
-    international: "Fast international payments"
-    internationaltext: "Bitcoins can be transferred from Africa to Canada in 10 minutes. In fact, bitcoins never have any real physical location, so it is possible to transfer any amount anywhere with no limits, delays, or excessive fees. There are no intermediate banks to make you wait three business days."
```

Comment:
> "10 minutes" (implies a first confirmation only takes 10 minutes, when they can take much longer)

But you can? If you know the current feeRate you can ensure (mostly with a fee less the 1USD) to get confirmed within the next block/~10min (at a chance above 95%).


Maybe we should also view the text from a advertising perspective. Let's don't be to particular about technical details: it's not a tech only documentation/description.